pom.xml

```diff
@@ -426,10 +472,18 @@
                                     <pattern>org.apache.http</pattern>
                                     <shadedPattern>${shadeBase}.org.apache.http</shadedPattern>
                                 </relocation>
+                                <relocation>
+                                    <pattern>mozilla</pattern>
```

Comment:
are these new dependencies from hadoop upgrade ? Seems like hadoop3 has lot of new dependencies.